docs/performance/getting-started.md

```diff
@@ -51,23 +51,27 @@ ngOnInit() {
 }
 ```
 
-### `trace(name:string)`
+### `trace(name, options?)`
```

Comment:
It wasn't clear to me that `options` was `TraceOptions`. What do you think about including `name: string` and `options?: TraceOptions` for these?